ttnn/cpp/ttnn/operations/matmul/device/matmul_op_multi_core_reuse_mcast_1d_program_factory.cpp

```diff
@@ -99,6 +100,11 @@ operation::ProgramWithCallbacks create_program_mcast_in0(
     if (B * num_blocks > 1) {
         in1_CB_tiles = in1_CB_tiles * 2;  // double buffer
     }
+    if (in1_is_sharded) {
+        uint32_t in1_shard_height_in_tiles = in1_buffer->shard_spec().shape()[0] / TILE_HEIGHT;
```

Comment:
After https://github.com/tenstorrent/tt-metal/pull/12908 you should avoid using TILE_HEIGHT/TILE_WIDTH